tests/integration/session/test_sign_out_and_exit.py

```diff
@@ -14,9 +14,18 @@
 
 
 class TestSaveAndSignOut(IntegrationTestCase):
```

Comment:
We should update `TestCensusSignOut` to ensure the sign-out for that doesn't use the `todo` path.

Reply:
Thanks, added new test in `TestCensusSignOut` that checks for that.